client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -10,26 +10,40 @@ import MediaBlock from '../blocks/MediaBlock';
  */
 const EmbedBlock = props => {
   const { entity, onRemoveEntity } = props.blockProps;
-  const { url, title, thumbnail } = entity.getData();
+  const { url, title, thumbnail, providerName, authorName } = entity.getData();
+
+  if (thumbnail) {
+    return (
+      <MediaBlock {...props} src={thumbnail} alt="">
+        {url ? (
+          <a
+            className="Tooltip__link EmbedBlock__link"
+            href={url}
+            title={url}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {title}
+          </a>
+        ) : null}
+
+        <button className="button button-secondary no Tooltip__button" onClick={onRemoveEntity}>
+          {STRINGS.DELETE}
+        </button>
+      </MediaBlock>
+    );
+  }
 
   return (
-    <MediaBlock {...props} src={thumbnail} alt="">
-      {url ? (
-        <a
-          className="Tooltip__link EmbedBlock__link"
-          href={url}
-          title={url}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {title}
-        </a>
-      ) : null}
-
-      <button className="button button-secondary no Tooltip__button" onClick={onRemoveEntity}>
+    <strong>
```

Comment:
I think I would still like this case to look visually similar to the `MediaBlock` component, perhaps overlaying the `providerName` & `authorName` text you have here on top of the existing block visual. Could you add a prop to `MediaBlock` for that, and then use the block with that extra prop?

Reply:
Applied your suggestions. Don't know how to approach it in terms of styling but we can discuss when you're back :)

Reply:
Preview looks like this

![Screenshot 2019-08-16 at 09 49 55](https://user-images.githubusercontent.com/52247241/63155932-86305b00-c00b-11e9-8489-3587368aa1b4.png)

Reply:
Update: Just noticed this doesn't work on Chrome